core/const.py

```diff
@@ -870,3 +870,31 @@ class ITLRegion(StrEnum):
         "RiskOwnerRole": "P{i}",
     },
 }
+
+# maps the financial year's start tear, or a month, back to original col letter
+# non-footfall outcomes are mapped by the financial year's start year, but footfall outcomes just by the month
+# TODO: add non-outcome tables to this
```

Comment:
Will it be possible to use the same dict for Finances and Outputs? or will we have duplicate keys? If we can't could rename this dict to specify it's only for Outcomes

Reply:
Possibly not, will have to investigate that. If not, will rename to make it clear it is exclusively a mapping for outcomes.